openapi/listings/listings.yaml

```
@@ -447,3 +571,299 @@ components:
         The listing score information for a business location
       x-tags:
         - Listing Scores
+    listingProfiles:
+      type: object
+      x-examples:
+        Example 1:
+          id: AG-1234567
+          type: businessLocations
+          attributes:
+            customerIdentifier: string
+            name: string
+            address:
+              line1: 109 8th Street E.
+              line2: Suite 23
+              city: string
+              stateCode: CA
+              zip: string
+              postalCode: S7M 1R3
+              regionCode: CA-SK
+              countryCode: CA
+            phoneNumbers:
+              - string
+            serviceAreaBusiness: true
+            geoCoordinate:
+              latitude: -90
+              longitude: -180
+            hours:
+              - hoursTypeId: general
+                regularHours:
+                  - openDay: wednesday
+                    closeDay: wednesday
+                    openTime: '09:45'
+                    closeTime: '09:45'
+                specialHours:
+                  - status: closed
+                    startDate: '2019-08-24'
+                    endDate: '2019-08-24'
+                    openTime: '09:45'
+                    closeTime: '09:45'
+            updatedAt: string
+          relationships:
+            businessPartner:
+              data:
+                type: partners
+                id: ABC
+            businessCategories:
+              data:
+                - type: businessCategories
+                  id: 'active:diving:freediving'
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Any entity that works with a provider company (two-way communication). These generally have a sustained relationship with the provider company. This may include a potential buyer, an existing client, or a past client that has churned. This entity should have Listing builder active in our platform
```

We should rewrite the description to be specific to listings. The first few lines are confusing. Maybe something like this?

```suggestion
        A listing profile is the representation of a business location (or similar) used by the Listing Builder product. It contains advanced fields used by listing sources. If the Listing Builder product is active for the location the data will be shared to listing sources such as Google Maps.
```